apps/calendar/js/views/month.js

```diff
@@ -120,9 +120,7 @@ Month.prototype = {
       this.currentFrame.deactivate();
     }
 
-    this._getFrame(this._previousTime());
```

Comment:
reduced to a single "frame" since now the busytimes are only loaded/rendered after the view is activated. gets some extra ms.